<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="PHX-ISHARAF" name="com.crm.qa.testcases.LogActivityTest" tests="24" failures="1" timestamp="27 Nov 2019 17:36:49 GMT" time="3604.366" errors="0">
  <testcase name="CreateRetail1Account" time="61.459" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reachedwithBusinessLead_UnplannedMeeting" time="79.827" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="CreateRetail1Account_Spouse" time="745.595" classname="com.crm.qa.testcases.LogActivityTest">
    <failure type="java.lang.AssertionError" message="timeout">
      <![CDATA[java.lang.AssertionError: timeout
	at org.junit.Assert.fail(Assert.java:88)
	at com.crm.qa.util.TestUtil.waitForElement(TestUtil.java:170)
	at com.crm.qa.pages.DetailsPage.enterComments(DetailsPage.java:475)
	at com.crm.qa.testcases.LogActivityTest.CreateRetail1Account_Spouse(LogActivityTest.java:533)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- CreateRetail1Account_Spouse -->
  <testcase name="CreateRetail1Account_Spouse" time="745.676" classname="com.crm.qa.testcases.LogActivityTest">
    <skipped/>
  </testcase> <!-- CreateRetail1Account_Spouse -->
  <testcase name="reachedwithBusinessLead" time="73.719" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="meeting_CompleteFlowWithOptions" time="109.603" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reached_Schedule_ToDo" time="84.726" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="meeting_CompleteFlow" time="112.280" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="UnplannedMeeting" time="84.942" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="notReached" time="78.216" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="UpdatePrimaryAccountdetails" time="118.733" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="CreateRetail2Account" time="78.932" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reachedwithBusinessLead_ScheduleTodo" time="95.626" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reached" time="69.704" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reachedwithBusinessLead_ScheduleMeeting" time="104.737" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="UpdateSpouseAccountdetails" time="118.423" classname="com.crm.qa.testcases.LogActivityTest">
    <skipped/>
  </testcase> <!-- UpdateSpouseAccountdetails -->
  <testcase name="LogaCallwithSpouse_ValidateAtPrimary" time="117.087" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reached_Schedule_Appointment" time="106.391" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reachedwithBusinessLead_SchedulePhoneCall" time="88.741" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="notReachedwithBusinessLead" time="75.276" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="reached_Schedule_Phonecall" time="86.413" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="LogACallSpouse" time="116.227" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="nolead_Reached_createBranchOpportunity" time="94.103" classname="com.crm.qa.testcases.LogActivityTest"/>
  <testcase name="UpdateSpouseAccountdetails" time="157.930" classname="com.crm.qa.testcases.LogActivityTest"/>
</testsuite> <!-- com.crm.qa.testcases.LogActivityTest -->
